app/models/content_item.rb

```diff
@@ -28,6 +28,15 @@ class ContentItem < ActiveRecord::Base
   NON_RENDERABLE_FORMATS = %w(redirect gone).freeze
   EMPTY_BASE_PATH_FORMATS = %w(government).freeze
 
+  has_one :state
+  has_one :location
+  has_one :translation
+  has_one :user_facing_version
+
```

Comment:
We've resisted adding the relationship accessors to ContentItem for fear of turning it into a "god object" (on the advice of @tuzz ). If there's a significant improvement in performance and readability as a result of doing this, then we can certainly discuss revisiting that decision.